_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -7,20 +7,42 @@ layout: page
 type: pages
 lang: en
 share: false
-version: 1
+version: 2
 ---
+{% include _toc.html %}
+
 This document describes the life-cycle of the Bitcoin Core software package released by the Bitcoin Core project. It is in line with standard maintenance policy across commercial software.  
 
-We aim to make a major release every 6-7 months, These will be numbered 0.11, 0.12 etc.
+## Versioning
+
+Bitcoin Core versions releases as follows 0.MAJOR.MINOR. Release candidates are suffixed with rc1, rc2 etc.
+
+## Major releases
+
+We aim to make a major release every 6-7 months.
+
+These will be numbered 0.11.0, 0.12.0 etc.
+
+## Maintenance releases
 
-We will provide maintenance "minor releases" that fix bugs within the major releases. We may add minor features to point releases where necessary, for example when back-porting consensus rule changes such as soft forks. Minor releases will be numbered 0.11.1, 0.11.2, 0.12.1, 0.12.2 etc.
+We will provide maintenance "minor releases" that fix bugs within the major releases. As a general rule we do not introduce major new features in a maintenance release (consensus rule are exempt from this rule). However, we may add minor features where necessary, and we will back-port consensus rule changes such as soft forks.
+
+Minor releases will be numbered 0.11.1, 0.11.2, 0.12.1, 0.12.2 etc.
+
+## Soft forks
```

Comment:
No reason to think the same wouldn't apply to hard forks as well (actually, it's even more important for HFs than SFs..). Maybe we should just refer to "consensus rule changes" here?